### PR TITLE
Remove old version of JanDeDobbeleer.OhMyPosh

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
null

### CODE REVIEW

File: manifests/t/Trufflesuite/Ganache/2.5.4/Trufflesuite.Ganache.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Trufflesuite.Ganache
+PackageVersion: 2.5.4
+PackageLocale: en-US
+Publisher: Trufflesuite
```
Comment (ItzLevvie) [inline manifests/t/Trufflesuite/Ganache/2.5.4/Trufflesuite.Ganache.locale.en-US.yaml:7]:
```suggestion
Publisher: Truffle
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Duplicate of https://github.com/microsoft/winget-pkgs/pull/25047.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/25064#pullrequestreview-730192062

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Incorrect `Publisher`.

**Comment by vedantmgoyal2009** [comment]
Setting Publisher to 'Truffle' leads to validation failure.

**Comment by ItzLevvie** [comment]
I don't know what the difference between this and https://github.com/microsoft/winget-pkgs/pull/25047 is, but could you please contribute over at https://github.com/OfficialEsco/winget-pkgs/tree/Trufflesuite.Ganache-2.5.4 instead?

It also looks like the `Installers:` section in file `Trufflesuite.Ganache.yaml` isn't meant to be there.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/25064#issuecomment-899398671

**Comment by vedantmgoyal2009** [comment]
After #25217 will get merged, my PR will pass the Validation Pipeline.

**Comment by ItzLevvie** [comment]
Since @OfficialEsco's PR was merged, this can be closed now.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/25064#issuecomment-900922819

**Comment by vedantmgoyal2009** [comment]
Please approve. I don't think it will create any issues.

**Comment by OfficialEsco** [comment]
Why?.. the only change is the location of the user scope

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/25064#issuecomment-900955678

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**